src/Atropos/Atropos.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DocumentationFile>C:\Users\jsmil\OneDrive\Documents\Software\Kvasir\src\Atropos\Intellisense.xml</DocumentationFile>
+    <DocumentationFile>..\Intellisense.xml</DocumentationFile>
```

Comment:
I think this should probably just be `.` because we want it in the same directory as the `.csproj` rather than one directory up. I'm pretty sure that's why we ended up with a new `Intellisense.xml` file that didn't exist before.

Reply:
(The same applies to the other files too)